_model_docs/relations.adoc

```diff
@@ -186,12 +186,18 @@ its own version of the textual content of the Work it is annotating.
 This is somewhat rare, and FRBR does not model annotations as
 distinct from revisions.
 
-draftOf:: document B is a specific unpublished or revision of document A;
+draftOf:: document B is a specific unpublished version or revision of document A;
 it may also be the first, unpublished
 expression of document A. Document A may be a Work or an Expression
 (e.g. a draft may be specific to an edition or translation).
 Document A need not itself be published. (_hasDraft_)
 
+preliminaryDraftOf::: document B is an unpublished version preceding and preparatory
```

Comment:
`preliminaryDraftOf` is meant as a "predecessor draft"?

`revisionDraftOf` is meant as a "successor draft"?

The wording is rather confusing and I have no idea what it means.

Reply:
Now at #77.